post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal user-provisioned infrastructures with `x86_64` control plane machines. 
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. Clusters with multi-architecture compute machines are available only on AWS or Azure installer-provisioned infrastructures and bare metal user-provisioned infrastructures with `x86_64` control plane machines.
+
+[IMPORTANT]
+====
+The Cluster Samples Operator is not supported on clusters with multi-architecture compute machines. Your cluster can be created without this capability. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc[Enabling cluster capabilities]
```

Comment:
@Prashanth684 Is this more what you were thinking? I personally am not too sure about it, it seems like it needs more info of some kind. Like a possible action to take